docs/extending/add_udw_tab.md

```diff
@@ -0,0 +1,167 @@
+- `image.js` renders a single image
+
+### `images.service.js`
+
+Create a service for fetching the images by adding `images.service.js` to `assets/js/image-tab/services/`:
+
+```js
+[[= include_file('code_samples/back_office/udw/assets/js/image-tab/services/images.service.js') =]]
+```
+
+### `images.list.js`
+
+Next, create an image list by adding an `images.list.js` to `assets/js/image-tab/components/`:
+
+```js
+[[= include_file('code_samples/back_office/udw/assets/js/image-tab/components/images.list.js') =]]
+```
+
+### `image.js`
+
+Finally, create an image view by adding an `image.js` to `assets/js/image-tab/components/`:
+
+```js
+[[= include_file('code_samples/back_office/udw/assets/js/image-tab/components/image.js') =]]
+```
+
+##  Add styles
+
+Ensure that the new tab is styled by adding the following files to `assets/css/`.
+
+### `images.list.css`
+
+```css
+[[= include_file('code_samples/back_office/udw/assets/css/image.list.css') =]]
+```
+
+### `image.css`
+
+```css
+[[= include_file('code_samples/back_office/udw/assets/css/image.css') =]]
+```
+
+### Add css to webpack
+
+Finally, add css in `webpack.config.js`:
+
+```js
+ibexaConfigManager.add({
+    ibexaConfig,
+    entryName: 'ibexa-admin-ui-layout-css',
+    newItems: [path.resolve(__dirname, './assets/css/image.css'), path.resolve(__dirname, './assets/css/images.list.css')],
+});
+```
+
+??? tip "Complete `webpack.config.js` code"
+
+    ```js
+    const Encore = require('@symfony/webpack-encore');
+    const path = require('path');
+    const getIbexaConfig = require('./ibexa.webpack.config.js');
+    const ibexaConfig = getIbexaConfig(Encore);
+    const customConfigs = require('./ibexa.webpack.custom.configs.js');
+    const ibexaConfigManager = require('./ibexa.webpack.config.manager.js');
+
+    Encore.reset();
+    Encore
+        .setOutputPath('public/build/')
+        .setPublicPath('/build')
+        .enableStimulusBridge('./assets/controllers.json')
+        .enableSassLoader()
+        .enableReactPreset()
+        .enableSingleRuntimeChunk()
+    
+    // Put your config here.
+    
+    ibexaConfigManager.add({
+        ibexaConfig,
+        entryName: 'ibexa-admin-ui-udw-tabs-js',
+        newItems: [path.resolve(__dirname, './assets/js/image-tab/image.tab.module.js')],
+    });
+    
+    ibexaConfigManager.add({
+        ibexaConfig,
+        entryName: 'ibexa-admin-ui-layout-css',
+    });
+    
+    // uncomment the two lines below, if you added a new entry (by Encore.addEntry() or Encore.addStyleEntry() method) to your own Encore configuration for your project
+    // const projectConfig = Encore.getWebpackConfig();
+    // module.exports = [ eZConfig, ...customConfigs, projectConfig ];
+    
+    // comment-out this line if you've uncommented the above lines
+    module.exports = [ ibexaConfig, ...customConfigs ];
+    ```
+
+## Check results
+    
+In the Back Office go to **Content** -> **Content structure**. On the left panel, click **Browse**.
```

Comment:
There is not more "Browse" in left panel, that's leftover from v3. The quickest way I can think of to get to UDW is to create an article and in Richtext field go to embed.